docs/Homebrew-on-Linux.md

```diff
@@ -85,8 +85,10 @@ Extract or `git clone` Homebrew wherever you want. Use `/home/linuxbrew/.linuxbr
 ```sh
```

Comment:
Let's just remove the entire `Alternative Installation` section; it's linked from the homepage `Install` notes anyway and exists mainly from when Linuxbrew had its own Homebrew/brew fork.